tsdb/engine/tsm1/file_store.go

```diff
@@ -449,6 +454,7 @@ func (f *FileStore) Close() error {
 		file.Close()
 	}
 
+	f.lastFileStats = nil
```

Comment:
This also needs to be done in `Replace` which is the primary way that files are added to the `FileStore` via compactions.